localstack/utils/common.py

```diff
@@ -15,18 +15,18 @@
 import tarfile
 import zipfile
 import binascii
-import calendar
 import tempfile
 import functools
 import threading
 import subprocess
+
 import six
 import shutil
 import requests
 import dns.resolver
 import platform
 from io import BytesIO
-from datetime import datetime, date
+from datetime import datetime, date, timezone
```

Comment:
I think we may need to import `timezone` in the function itself, as this may break some Python 2.x Lambdas which are importing this code (not ideal, we could look into fixing this properly at some point..).

Reply:
yep i figured as much and working on it :-)